src/main/java/com/puppycrawl/tools/checkstyle/checks/naming/AbbreviationAsWordInNameCheck.java

```diff
@@ -93,12 +93,69 @@
  * </li>
  * </ul>
  * <p>
- * Default configuration
+ * Default Configuration:
```

Comment:
Usually it says `To configure the check:`

Reply:
Fixed. Thanks!